scripts/initializers/search.js

```diff
@@ -2,15 +2,23 @@ import { initializers } from '@dropins/tools/initializer.js';
 import {
   initialize,
   setFetchGraphQlHeaders,
+  getFetchGraphQlHeader,
   setEndpoint,
 } from '@dropins/storefront-product-discovery/api.js';
 import { getHeaders } from '@dropins/tools/lib/aem/configs.js';
 import { initializeDropin } from './index.js';
 import { fetchPlaceholders, commerceEndpointWithQueryParams } from '../commerce.js';
 
 await initializeDropin(async () => {
-  setEndpoint(await commerceEndpointWithQueryParams());
-  setFetchGraphQlHeaders((prev) => ({ ...prev, ...getHeaders('cs') }));
+  // Set Fetch Headers (Service)
+  let customerGroupHeader = getFetchGraphQlHeader('Magento-Customer-Group');
+  customerGroupHeader = customerGroupHeader ? { 'Magento-Customer-Group': customerGroupHeader } : {};
```

Comment:
Here, and in other initializers, we are overloading this variable. They are basically free, so may as well create another. Also newlines are basically free as well :)

```suggestion
  const customerGroupHeaderValue = getFetchGraphQlHeader('Magento-Customer-Group');
  const customerGroupHeader = customerGroupHeaderValue ? { 
    'Magento-Customer-Group': customerGroupHeaderValue 
   } : {};
```

Reply:
🤣 I don't really like both: too much new lines and too much disposable variables.